docs/guide/0-values.md

```diff
@@ -1,25 +1,29 @@
 
-## 1. We optimize for iterating quickly
+## 2 — Do not work on something until you understand why you are working on it
 
-We aren’t trying to build software around a future we can’t predict. We don’t write code that we may not need. We know that no abstractions are better than bad abstractions.
+We can only reflect on the impact of our work if we can explain why we are doing it. Understanding what guides our efforts is important to remain focused. Reality tends to move, and focus needs to be corrected. Thoughtless inertia can leave us adrift. Without clear intent, motion masquerades as progress.
 
-## 2. We are here for the long haul
+## 3 — Do not mistake shortcuts for cutting the journey short
 
-We understand that Calypso is here to stay and we put the extra effort into its design. We regularly act to make _the design simpler_ and more consistent. We invest in deep understanding of the tools, languages, and techniques we use. We learn constantly.
+Every shortcut promises a faster arrival but could end somewhere else. True velocity comes from moving quickly on the right path. Instead, we rationalize detours as worthwhile iterations. The person that takes one step at a time on the correct path will eventually arrive at their destination. The person looking for constant shortcuts to avoid the effort of walking can easily become stranded. All too often we rationalize incomplete solutions as stepping stones, when they’re really diverging paths. True progress requires solving core problems, not circumventing them. Bad shortcuts are often dead ends.
 
-## 3. We don’t trust ourselves to be perfect
+## 4 — Do not glorify failure under the guise of learning
 
-We get the product to our colleagues and users as early as possible. We review each other’s code. We show incomplete work and seek critical feedback. We let automated tools help us with tasks from checking coding standards to running tests. We keep an open mind and a beginner mindset.
+There is a difference between the practice of improving based on correcting mistakes and the gradual indifference that settles when we become used to them. It is not uncommon for bad decisions to go unexamined. Those who never make mistakes will hardly improve, but those who internalize mistakes are sure to keep repeating them. It’s the difference between learning from mistakes and making mistakes a learned habit. Excellence is not an act but a practice.
 
-## 4. We are in it together
+## 5 — Work is invisible until it is real
 
-We help each other, because no team or person can succeed if Calypso doesn’t succeed. We give candid feedback, regardless of hierarchy. We don’t own the code we write and the features we build. We are not afraid to work on areas outside of our direct responsibilities. We leave our egos at the door.
```

Comment:
Some of the removed principles might still be valuable and part of our main principles, even if they're not part of Matias's prior article. Have you considered leaving some of these at the end of the document?

Reply:
I do agree they are valuable, so is the creed for instance but I think it's better to keep a consistent message and keeping different things kind of dilutes the message. 

I even considered removing this file entirely.

Reply:
Leaving it to your judgment. I think the philosophical parts may belong better in other places, and the technical, best practice, and preferred stack docs are way more practical and useful for folks in general.